pages/myProfile/userCounter/userCounter.tsx

```diff
@@ -0,0 +1,15 @@
+import s from './userCounter.module.scss';
+
+type UserCounterType = {
+  className?: string;
+  count: number;
+  text: string;
+};
+export const UserCounter = (props: UserCounterType) => {
+  return (
+    <div className={props.className}>
+      <div className={s.userCounter}>{props.count} </div>
```

Comment:
It should be replaced by the `Typography` component, which is designed to handle phrasing content on application pages.